todo_app/app.py

```diff
@@ -1,23 +1,81 @@
 from flask import Flask, render_template, request, redirect
-from todo_app.data.session_items import get_items, add_item
+from todo_app.todo import Item
 
 from todo_app.flask_config import Config
+import requests
+import os
 
 app = Flask(__name__)
 app.config.from_object(Config())
 
 
 @app.route('/')
 def index():
-    items = get_items()
+     
+    url = "https://api.trello.com/1/boards/vwBBuYdA/lists"
```

Comment:
You should use your environment variables:
```suggestion
    url = f"https://api.trello.com/1/boards/{os.getenv('TRELLO_BOARD_ID')}/lists"
```

Note the `f` at the start to make it a format string, and the single quotes for the inner bit so Python doesn't think it's the end of the actual string.

Reply:
Thanks, Hugh. I've made these changes and merged into the main.